src/main/resources/application.properties

```diff
@@ -110,9 +110,12 @@ management.endpoint.metrics.cache.time-to-live=1000ms
 
 logging.level.uk.gov.hmcts.ccd.domain.service=${DATA_STORE_DEFAULT_LOG_LEVEL:INFO}
 logging.level.uk.gov.hmcts.ccd=${DATA_STORE_DEFAULT_LOG_LEVEL:INFO}
-# logging.level.org.springframework.web=DEBUG
-#logging.level.org.hibernate=INFO
-#logging.level.org.hibernate.type.descriptor.sql=trace
+
+logging.level.org.springframework.web=DEBUG
+logging.level.org.hibernate=INFO
+logging.level.org.hibernate.type.descriptor.sql=trace
+logging.level.org.apache.http=DEBUG
```

Comment:
Isn't this too much logging? Is it just for testing? Also how is the hibernate logging level related to this change?